cfgov/core/migrations/0001_initial.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+from django.contrib.contenttypes.models import ContentType
+
+
+def remove_reversion_ct(apps,schema_editor):
+    ContentType.objects.filter(app_label='reversion').delete()
```

Comment:
Can you refer to a model via import like this, in a migration? I think you might need to do something like

```py
ContentType = apps.get_model('contenttypes', 'ContentType')
```

to void errors.

Reply:
hmm, it worked for me-- but this is probably cleaner.

Reply:
Right, it'll work, but generally best practice in Django migrations is not to import models like that. The problem would be, if, say, in a future Django version the `ContentType` model has some different fields. This migration might run before the one modifying `ContentType`, which means that the state of the model via import from code would differ from the one in the DB. Using `apps.get_model` ensures that you always get the right model version at the time of the migration.